### PR TITLE
feat(24.04): add ffmpeg dependencies slices

### PR DESCRIPTION
**Pull request by HadrienPatte**
# Proposed changes

The overall goal is to create a slice for `ffmpeg`. Since `ffmpeg` has a pretty deep dependency tree, this will be broken up in a series of smaller PRs. This first PR adds SDF for the following packages that are all transitive dependencies of `ffmpeg`:
* libaom3
* libapparmor1
* libasyncns0
* libblkid1
* libcjson1
* libmp3lame0
* ibopus0
* libpixman-1-0
* libraw1394-11
* libsamplerate0
* libudfread0
* libunibreak5
* libva2
* libvpx9
* libwayland-egl1
* libx264-164
* libxau6
* libxshmfence1
* libxvidcore4
* libzix-0-0

## Testing

```bash
# Example:
#  chisel cut ... --root <path> <pkg>_<proposed-slice>
#  sudo chroot <path> <app>
```

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: slices/libx264-164.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+essential:
+  - libx264-164_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libx264.so.164*:
```
Comment (rebornplusplus) [inline slices/libx264-164.yaml:11]:
```suggestion
      /usr/lib/*-linux-*/libx264.so.164:
```
Wildcard not needed since there is only one such file.

**Review by rebornplusplus — COMMENTED** [review]
Looks good to me overall, thank you!

Only left a very minor change request.

**Comment by rebornplusplus** [comment]
I see the CLA check is failing. You may need to sign the [CLA form](https://ubuntu.com/legal/contributors/agreement) with appropriate email.

File: slices/libx264-164.yaml (hunk 1 above)
Comment (HadrienPatte) [reply slices/libx264-164.yaml]: Thanks for the review, updated the PR accordingly and  signed the CLA

**Comment by HadrienPatte** [comment]
@rebornplusplus I have signed the CLA again, should be good now

**Comment by linostar** [comment]
@letFunny @cjdcordeiro can you approve the workflow please?

**Comment by HadrienPatte** [comment]
@cjdcordeiro gentle bump :)

**Review by rebornplusplus — APPROVED** [review]
Looks all good to me, thank you!

(Sorry for the delay in re-review, I was OOO for a while.)

**Review by linostar — APPROVED** [review]
LGTM, thanks!

**Review by cjdcordeiro — APPROVED** [review]
Y lgtm too thanks!

**Pull request merged**